deploy/osd-ingress/config.yaml

```diff
@@ -0,0 +1,3 @@
+deploymentMode: "SelectorSyncSet"
+selectorSyncSet:
+    resourceApplyMode: Upsert
```

Comment:
@cblecker Use Upsert here, so the openshift-sre-ssh ns removed here will not delete the ns in OSD.

Reply:
We want this NS deleted on everything.. *except* hive clusters